Sources/HttpServer/Route/Route.swift

```diff
@@ -2,10 +2,26 @@ public class Route {
     public var url: String
     public var method: HttpMethod
     public var action: HttpAction
+    public var realm: String?
+    public let authorization: String?
 
-    public init(url: String, method: HttpMethod, action: HttpAction) {
+    public init(url: String, method: HttpMethod, action: HttpAction, realm: String? = nil, authorization: String? = nil) {
```

Comment:
Why are these set to nil by default? Is there a better way to do this rather than setting nils?

Reply:
because realm and authorization fields are not required, right?